### ISSUE TITLE
Skip first H1 headline is broken

### ISSUE THREAD

**Issue by dotmagic** [jonschlinkert/markdown-toc#42]
Generate TOC with the option `firsth1:false` outputs undefined. Btw, I would also suggest to use a more meaningful name like `skipFirstHeadline`

**Comment by jonschlinkert** [comment]
Can you show me where that is documented? The option is `firsth1` I believe. [documented here]( https://github.com/jonschlinkert/markdown-toc#optionsfirsth1)

**Comment by jonschlinkert** [comment]
You can also use [options.filter](https://github.com/jonschlinkert/markdown-toc#optionsfilter) to exclude headings.

**Comment by dotmagic** [comment]
Yes. I did use `firsth1` exactly as documented. The first Headline isn't displayed but undefined. From a quick look at the source it seems this option wasn't updated with code changes.

**Comment by jonschlinkert** [comment]
ah, ok thanks so much for double-checking. looks like a bug
